plugins/inputs/jboss/jboss.go

```diff
@@ -0,0 +1,1312 @@
+package jboss
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	dac "github.com/stefa975/go-http-digest-auth-client"
```

Comment:
Any reason you don't use the "upstream" version at https://github.com/xinsnake/go-http-digest-auth-client?